packages/react/src/LayoutExamples/KitchenSink.tsx

```diff
@@ -28,26 +28,38 @@ type KitchenSinkProps = React.HTMLAttributes<HTMLDivElement>
 
 export function KitchenSink(props: KitchenSinkProps) {
   const [enableGridOverlay, setEnableGridOverlay] = React.useState(false)
+  const [isLightMode, setIsLightMode] = React.useState(true)
 
   const handleOverlay = e => {
     e.preventDefault()
     setEnableGridOverlay(!enableGridOverlay)
   }
 
+  const handleMode = e => {
+    e.preventDefault()
+    setIsLightMode(!isLightMode)
+  }
+
   return (
-    <>
+    <ThemeProvider
+      colorMode={isLightMode ? 'light' : 'dark'}
+      style={{backgroundColor: 'var(--brand-color-canvas-default)'}}
+    >
       <SubdomainNavBar title="Subdomain" {...props}>
         <SubdomainNavBar.Link href="#">Collections</SubdomainNavBar.Link>
         <SubdomainNavBar.Link href="#">Topics</SubdomainNavBar.Link>
         <SubdomainNavBar.Link href="#">Articles</SubdomainNavBar.Link>
         <SubdomainNavBar.Link href="#">Events</SubdomainNavBar.Link>
         <SubdomainNavBar.Link href="#">Video</SubdomainNavBar.Link>
         <SubdomainNavBar.Link href="#">Social</SubdomainNavBar.Link>
+        <SubdomainNavBar.SecondaryAction href="#" onClick={handleMode}>
+          Change color mode
+        </SubdomainNavBar.SecondaryAction>
         <SubdomainNavBar.PrimaryAction href="#" onClick={handleOverlay}>
           {enableGridOverlay ? 'Disable' : 'Enable'} grid
         </SubdomainNavBar.PrimaryAction>
       </SubdomainNavBar>
-      <Grid style={{margin: '80px auto 200px'}} enableOverlay={false}>
+      <Grid style={{padding: 'var(--base-size-80) auto 200px'}} enableOverlay={false}>
```

Comment:
Smoke tested this, I think the padding is not necessary since the Grid has a `max-width` and the `margin: 0 auto` 

```suggestion
      <Grid enableOverlay={false}>
```

Reply:
Thanks for spotting this. It wasn't applying correctly before because `auto` isn't a valid value. Updated to `0` and it's now working as expected. The inline style (as ugly as it is) takes a specificity precedence over the Grid default styles. That's why it's here.